services/productService.js

```diff
@@ -0,0 +1,26 @@
+const productDao = require("../models/productDao");
+
+const getProducts = async (categoryId, sort) => {
+  try {
+    return productDao.getProducts(categoryId, sort);
+  } catch (err) {
+    const error = new Error("INVALID_DATA");
+    error.statusCode = 500;
```

Comment:
```suggestion
    error.statusCode = 400;
```
500이라는 상태코드 자체는 웹 사이트 서버에 문제가 있음을 의미하지만, 보통 서버에서 의도치 않은 예외 혹은 에러에 대한 상태코드가 500으로 반환되기 때문에 지금처럼 `"INVALID_DATA"` 이라는 메세지를 부여하고 처리하고 있는 경우에는 `400`과 같은 코드를 사용해주세요.